README.md

```diff
@@ -1,5 +1,6 @@
-# Wikimedia Commons Android app [![Build status](https://api.travis-ci.org/commons-app/apps-android-commons.svg?branch=master)](https://travis-ci.org/commons-app/apps-android-commons)
-
+# Wikimedia Commons Android app
+[![Build status](https://api.travis-ci.org/commons-app/apps-android-commons.svg?branch=master)](https://travis-ci.org/commons-app/apps-android-commons)
+[![Preview the app](https://img.shields.io/badge/Preview-Appetize.io-orange.svg)](https://appetize.io/app/8ywtpe9f8tb8h6bey11c92vkcw)
```

Comment:
I think it would be nice to keep the badges just near the title or they should be separated from the description by adding a new line after the badges.

It [doesn't render well](https://github.com/madhurgupta10/apps-android-commons/tree/issue-2842#wikimedia-commons-android-app) as of now.

Reply:
@sivaraam Sure :)